rpc/README.md

```diff
@@ -4723,6 +4772,18 @@ The 10-byte fixed-length binary encoded as a 0x-prefixed hex string in JSON.
 
 
 
+### Type `RawTxPool`
+
+All transaction in tx-pool
+
+Verbose True for a json object, false for array of transaction ids
```

Comment:
`True` -> `true`.

By the way, could `verbose` be an enumerated type?
So I can dump the transaction pool in different modes:
- Hashes only.
- Hashes with `cycles`, `size` and `fee`.
- Whole transactions with `cycles`, `size` and `fee`.